noark-extraction-validator/src/main/java/com/documaster/validator/converters/xsd/XsdConverter.java

```diff
@@ -275,6 +276,10 @@ private void addFieldsOfClassToItemDef(Class<?> cls, ItemDef itemDef, Map<String
 				// Basic Java type
 				itemDef.addField(field.getName(), Field.FieldType.getFromJavaType(fieldCls));
 
+			} else if (field.getName().equalsIgnoreCase("systemid")) {
+
+				// Otherwise it would be detected as a reference
```

Comment:
I am not fond of this explicit handling by field name in a class that is quite generic. However, I don't currently have any other suggestions either.

Let's add a comment that this applies to the Noark 5.5 command only for now.

I will write an additional comment if I think this can be changed.

Reply:
I don't think I have any other suggestions for now.